mutmut/node_mutation.py

```diff
@@ -125,6 +128,62 @@ def operator_string_methods_swap(
             func_name = cst.ensure_type(node.func, cst.Attribute).attr
             yield node.with_deep_changes(func_name, value=new_call)
 
+def unsymmetrical_string_methods_swap(
+    node: cst.Call
+) -> Iterable[cst.Call]:
+    """Try to handle specific mutations of string, which useful only in specific args combination."""
+
+    for old_call, new_call in supported_unsymmetrical_str_methods_swap:
+        if m.matches(node.func, m.Attribute(value=m.DoNotCare(), attr=m.Name(value=old_call))):
```

Comment:
libcst does not verify the properties that we do not mention, so the `value=m.DoNotCare()` is superfluous.

```suggestion
        if m.matches(node.func, m.Attribute(attr=m.Name(value=old_call))):
```